pmd-core/etc/xslt/pmd-report-v2.xslt

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+xmlns:scan="http://pmd.sourceforge.net/report/2.0.0"
+  >
+
+<xsl:template match="scan:pmd">
+  <xsl:variable name="version" select="@version" />
+  <xsl:variable name="timestamp" select="@timestamp" />
+  <xsl:variable name="count">0</xsl:variable>
+  <xsl:variable name="total">0</xsl:variable>
+
+
+  <html>
+   <head>
+   <title>PMD Report </title>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous"/>
```

Comment:
How should we deal with updates of these libraries? E.g. bootstrap 5.2.1 is now available.
I think, whatever version we use here, it will be outdated very soon. Is this a problem we need to address?